tests/data/wf6-composite-inputs.gxwf-test.yml

```diff
@@ -0,0 +1,11 @@
+- doc: Test Composite Inputs
+  job:
+    input1:
+      class: File
+      filetype: imzml
+      composite_data:
+        - Example_Continuous.imzML
```

Comment:
Should that be `location: URL` or `path: Example_Continuous.imzML`?

Reply:
Going to leave this as an open issue - it is an important issue though and needs to be done.